app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -20,12 +20,13 @@ import cats.data.NonEmptyList
 import julienrf.json.derived
 import play.api.data.validation.ValidationError
 import play.api.libs.json._
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.DisplayWidthAttribute.DisplayWidthAttribute
 
 import scala.collection.immutable._
 
 sealed trait ComponentType
 
-case class Text(constraint: TextConstraint, value: Expr) extends ComponentType
+case class Text(constraint: TextConstraint, value: Expr, displayWidth: DisplayWidthAttribute) extends ComponentType
```

Comment:
Would it be appropriate to add default value for displayWidth, do you think?  That would reduce the extra verbosity in many of the unit tests and reduce the overall size of the change

Reply:
yes that's a good idea